app/routes/bestuurseenheid/zitting/besluitenlijst/index.js

```diff
@@ -1,24 +1,33 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default class BestuurseenheidZittingBesluitenlijstIndexRoute extends Route {
-  breadCrumb = { title: 'Besluitenlijst' };
+  @service store;
 
   async model() {
-    let zitting = this.modelFor('bestuurseenheid.zitting');
+    let zitting = await this.modelFor('bestuurseenheid.zitting');
     let besluitenlijst = await zitting.besluitenlijst;
     let besluiten = await this.store.query('besluit', {
       page: {
         number: 0,
-        size: 100
+        size: 100,
       },
-      "filter[besluitenlijst][:id:]": besluitenlijst.id,
-      sort: "volgend-uit-behandeling-van-agendapunt.position",
+      'filter[besluitenlijst][:id:]': besluitenlijst.id,
+      sort: 'volgend-uit-behandeling-van-agendapunt.position',
+      include: 'volgend-uit-behandeling-van-agendapunt',
     });
 
+    const bestuursorgaan = await zitting.bestuursorgaan;
+    await bestuursorgaan.isTijdsspecialisatieVan;
+
+    for (let i = 0; i < besluiten.length; i++) {
+      await besluiten.objectAt(i).volgendUitBehandelingVanAgendapunt;
```

Comment:
Is this needed? shouldn't it already be part of the other request due to the include?

Reply:
Not needed anymore, removed.